packages/riipen-ui/src/components/RadioButton.test.jsx

```diff
@@ -0,0 +1,154 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import { faFlag } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import RadioButton from "./RadioButton";
+
+const icon = i => <FontAwesomeIcon icon={i} />;
+
+describe("<RadioButton>", () => {
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(<RadioButton />);
+    } catch (theError) {
+      error = theError;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(<RadioButton />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("checked prop", () => {
+    it("sets class name as checked and color as initial when checked is true", () => {
+      const checked = true;
+
+      const wrapper = mount(<RadioButton checked={checked} readOnly />);
+
+      expect(
+        wrapper
+          .find("RadioButton")
+          .childAt(0)
+          .hasClass("checked")
+      ).toEqual(true);
+      expect(wrapper.find("Typography").props().color).toEqual("initial");
+    });
+
+    it("sets color as grey600 when checked is false", () => {
+      const checked = false;
+
+      const wrapper = mount(<RadioButton checked={checked} readOnly />);
+
+      expect(wrapper.find("Typography").props().color).toEqual("grey600");
+    });
+  });
+
+  describe("classes prop", () => {
+    it("sets custom class", () => {
+      const classes = ["newClass"];
+
+      const wrapper = mount(<RadioButton classes={classes} />);
+
+      expect(
+        wrapper
+          .find("RadioButton")
+          .childAt(0)
+          .hasClass(classes[0])
+      ).toEqual(true);
+    });
+
+    it("appends higher order values to classes prop with withClasses decorator", () => {
+      const classes = ["riipen", "riipen-radiobutton"];
+
+      const wrapper = mount(<RadioButton />);
+
+      expect(
+        wrapper
+          .find("RadioButton")
+          .props()
+          .classes.sort()
+      ).toEqual(classes.sort());
+    });
+  });
+
+  describe("disabled prop", () => {
```

Comment:
Same question as checked prop. Disabled prop is also used in the label and input elements.